integrations/hardhat/README.md

```diff
@@ -4,7 +4,7 @@ A plugin for Hardhat that wraps the provider connected to a Sapphire network.
 
 ## Usage
 
-First install the plugin. Make sure that `hardhat` is also installed.
+First install the plugin. Make sure that `hardhat` is not installed so that this plugin can install the oasis version of hardhat.
```

Comment:
No, this shouldn't be a requirement. It should be a literal drop-in replacement. The dependencies don't conflict due to the way npm works, so this is an unnecessary requirement.